lenovo/thinkpad/e495/default.nix

```
@@ -0,0 +1,10 @@
+{ config, lib, pkgs, ... }:
+
+{
+  imports = [
+    ../.
+    ../../../common/cpu/amd
+  ];
+
+  boot.kernelPackages = pkgs.linuxPackages_5_2;
```

To make this more future proof:

```suggestion
  boot.kernelPackages = lib.mkIf (lib.versionOlder linux.version "5.2") pkgs.linuxPackages_latest;
```

---

This is untested!